samples/minimal-samples/powermax.yaml

```diff
@@ -7,3 +7,6 @@ spec:
   driver:
     csiDriverType: "powermax"
     configVersion: v2.12.0
+  modules:
+    - name: resiliency
+      enabled: true
```

Comment:
Do we want to enable the resiliency module by default? If yes, can this behavior be made default and avoid these params in the minimal sample?

Reply:
nope we will not enable by default, The user has to give it explicitly. Let me add that in the comment